CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ v1.8.0 (unreleased)
     if the response body was empty.
 -   Adds support for the `UnrecognizedProcedureError` error and error checker,
     indicating that the router was unable to find a handler for the request.
+-   Exposes a `InboundMiddleware` function on the dispatcher to allow applying
+    middleware to custom handlers.
```

Comment:
What kind of custom handlers? I would like to be sure that exposing the inbound middleware accessor is a better solution than subsuming custom handlers behind an API that applies the middleware internally.